README.md

```diff
@@ -276,6 +272,68 @@ We do have already a couple of ideas:
 
 But please share your thoughts.
 
+## Limit
+
+This module is based on SQLite so it has all the SQLite strenghts and limitations.
+
+The appropriate use cases for SQLite are described in [this document](https://sqlite.org/whentouse.html).
+
+With this module we remove the network limitation and so the use of this module is not suggested in only two cases:
+
+#### Many concurrent writers.
+
+SQLite does hold a table level lock on write while "standard" database can hold a row, or even value level lock.
+
+This means that concurrent client will never be able to write at the same time on the same table and one will always need to wait for the other to finish.
+
+At the moment this limiting factor is only secondary with the respect of this module.
+Indeed, the module, is not multithread (it will be soon, though), so before to blame SQLite for slow concurrent write you must blame me, the author.
+
+However, it should not be an issues for most uses cases, if your specific use case require a lot of concurent read I would suggest you to still try and benchmark this implementation.
+
+#### BIG dataset
+
+Because its internal SQLite can handle only up to 140TB of data, ideally this will also apply to this same module, supposing you know where to host the database.
```

Comment:
Clarify that you hit the disk since Redis' major claim to fame is being in memory and SQLite supports in-memory modes for temporary dbs.